h/views/client.py

```diff
@@ -132,6 +176,20 @@ def absolute_asset_urls(bundle_name):
     }
 
 
+@view_config(route_name='embed',
+             has_feature_flag='use_client_boot_script',
+             http_cache=60 * 5)
```

Comment:
We want this to be cached by the CDN, so need to specify the `public` C-C token: `http_cache=(300, {'public': True})` as with robots.txt.